LayoutTests/platform/mac-wk1/TestExpectations

```diff
@@ -1585,6 +1585,9 @@ fast/dom/HTMLAnchorElement/anchor-file-blob-convert-to-download.html [ Skip ]
 fast/dom/HTMLAnchorElement/anchor-file-blob-convert-to-download-async-delegate.html [ Skip ]
 http/tests/download/convert-cached-load-to-download.html [ Skip ]
 
+# application/octet-stream isn't useable in WK1
```

Comment:
It is in some cases.  Is it not useable as the main resource?

Reply:
Yeah, I think it was main resources because we call `WebView _canHandleRequest` in decidePolicyForNavigationAction, and that doesn't know how to handle application/octet-stream.